src/themes/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from 'react-native'
+
+const StyleUtils = StyleSheet.create( {
```

Comment:
I would not CamelCase this, since it's an instance of something (`StyleSheet.create()`), so is effectively just a standard variable